ga4gh/serverconfig.py

```diff
@@ -84,3 +86,29 @@ class TestOidcConfig(TestConfig):
     OIDC_AUTHZ_ENDPOINT = "https://accounts.example.com/auth"
     OIDC_TOKEN_ENDPOINT = "https://accounts.example.com/token"
     OIDC_TOKEN_REV_ENDPOINT = "https://accounts.example.com/revoke"
+
+
+class FlaskDefaultConfig(object):
```

Comment:
This is a bit ugly --- is there any other way to reset to a Flask default? If we keep this we should add some comments explaining that it's only used in testing.